### PR TITLE
Re-structure issue template

### PR DESCRIPTION
**Pull request by pavlenex**
This PR is an effort to better structure our issues.
The idea is to separate questions, discussions, feature requests and chit-chat from actual `Issues` that we want tackled.
Issues should be a to-do actionable tasks that we want to tackle. Tasks that are generally accepted by contributors.
Discussions are place to ask questions, suggest a feature where community can discuss and vote on it. 

Now, what this PR does is structures the "Issues page" a bit better so it provides better context on what can be found where when a new user wants to report something.

I've also deleted feature requests template as I don't think any random idea should be in the issues. Instead now discussions mention that feature requests go there. There's still a lot of stuff that have to be done in terms of structuring discussions and moving issues towards discussions. 

## Before
BTCPayIssuesNow

## After
BTCPayIssuesNow2

Unfortunately Security.md template is hard-coded into issue template so couldn't add an emoji.

## Bug report template

I also made minor adjustments to the bug report template, mostly adding `<!-- detailed explanation that won't be visible in the issue when previewing so it does not clutter stuff -->`

## Bug report for the person reporting (adds context)
bbb

## Bug report when filed (cleaner without context)
aaaa
 
Live test is available on my fork here https://github.com/pavlenex/btcpayserver/issues/new/choose 

Please test, suggest improvements and let me know if you like the concept.

### CODE REVIEW

**Comment by Zaxounette** [comment]
ACK x10000

Love those improvements ! 

This should hopefully redirect support 100% to the chat instead of having support issues popping up from time to time.

Also, the context text is a nice idea. Anyone able and willing to report a bug should not be confused by it.

**Comment by pavlenex** [comment]
I am just unsure if I should communicate `Feature Request` in `Discussion` a bit more clearer. If anyone has a better idea of text / heading for `Discussions` feel free to suggest.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 blank_issues_enabled: false
-contact_links:
-  - name: Community Support Chat 
+contact_links: 
+  - name: 🚀 Discussions  
     url: https://chat.btcpayserver.org/
```
Comment (Kukks) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://github.com/btcpayserver/btcpayserver/discussions
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
-    about: Ask general questions and get community support in real-time.
+    about: Technical discussions, questions and feature requests    
+  - name: 📝 Official Documentation
+    url: https://github.com/btcpayserver/btcpayserver/discussions
```
Comment (Kukks) [inline .github/ISSUE_TEMPLATE/config.yml:7]:
```suggestion
    url: https://docs.btcpayserver.org
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -1,38 +1,63 @@
...
+
+Please fill in as much of the template below as you're able. 
+-->
+
 **Describe the bug**
 A clear and concise description of what the bug is.
 
-**To Reproduce**
-Steps to reproduce the behavior:
+**To Reproduce a bug**
```
Comment (Kukks) [inline .github/ISSUE_TEMPLATE/bug-report.md:20]:
```suggestion
**To Reproduce the bug**
```

**Review by Kukks — CHANGES_REQUESTED** [review]
nicer for sure

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (pavlenex) [reply .github/ISSUE_TEMPLATE/config.yml:7]: omg , thanks for catching this one.

**Review by Kukks — APPROVED** [review]

**Pull request merged**